sphinx-doc/conf.py

```diff
@@ -28,7 +28,7 @@
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
-    'numpy': ('https://docs.scipy.org/doc/numpy', None),
+    'numpy': ('https://numpy.org/doc/stable', None),
```

Comment:
NumPy moved their docs URL. This hides a warning that pops up when building the docs.